docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+
+.. _method_documentation:
+
+======
+Method
+======
+
+While **mrsimulator**'s organization of the :ref:`spin_sys_api` object and its
+composite objects, :ref:`site_api`, and :ref:`coupling_api` are easily
+understood by anyone familiar with the underlying physical concepts, the
```

Comment:
@pjgrandinetti Change the phrasing to "familiar with the underlying physical concepts **of NMR**"?

Reply:
doubt the user would be reading about writing a Method if they weren't already familiar with NMR.